contribute/general/quick-start-with-vim-go.md

```diff
@@ -2,7 +2,7 @@
 
 ### Pre-requisites
 
-Let us assume that you already have a working Go environment. You to make sure that your `$GOPATH/bin` is included in your PATH.
+Let us assume that you already have a working Go environment. You need to make sure that your `$GOPATH/bin` is included in your PATH.
```

Comment:
This could be  an enhancement but a lot of time `$GOPATH` is not set so if we could add a way to verify it that would be helpful.